spec/request/swagger_spec.rb

```diff
@@ -517,6 +517,13 @@
         end
       end
 
+      it 'supports getting separation_locations' do
+        expect(subject).to validate(:get, '/v0/disability_compensation_form/separation_locations', 401)
```

Comment:
@SarahJaine What I missed earlier when we were talking was the `headers` argument.  The 401 response is unauthenticated.